src/common/config_opts.h

```diff
@@ -881,6 +881,9 @@ OPTION(osd_deep_scrub_interval, OPT_FLOAT, 60*60*24*7) // once a week
 OPTION(osd_deep_scrub_randomize_ratio, OPT_FLOAT, 0.15) // scrubs will randomly become deep scrubs at this rate (0.15 -> 15% of scrubs are deep)
 OPTION(osd_deep_scrub_stride, OPT_INT, 524288)
 OPTION(osd_deep_scrub_update_digest_min_age, OPT_INT, 2*60*60)   // objects must be this old (seconds) before we update the whole-object digest on scrub
+OPTION(osd_deep_scrub_large_omap_object_key_threshold, OPT_U64, 2000000)   // threshold for number of keys to determine a large omap object
+// threshold for summed size (bytes) of all key values to determine a large omap object 1GB
+OPTION(osd_deep_scrub_large_omap_object_value_sum_threshold, OPT_U64, 1*1024*1024*1024)
```

Comment:
because existing bucket indexes would already exceed the lower osd_max_object_size default (128MB = ~500k bucket index entries), and I'd rather not raise the osd_max_object_size